actionpack/lib/action_controller/metal/redirecting.rb

```diff
@@ -87,7 +87,9 @@ def redirect_to(options = {}, response_status = {})
     #
     # All other options that can be passed to <tt>redirect_to</tt> are accepted as
     # options and the behavior is identical.
-    def redirect_back(fallback_location:, allow_other_host: true, **args)
+    def redirect_back(fallback_location: nil, allow_other_host: true, **args, &block)
+      fallback_location ||= block
+      raise 'fallback_location or block is required' unless fallback_location
```

Comment:
This should probably raise a `ActionControllerError` instead of a runtime error.

Reply:
ArgumentError fits better. Do you agree?

Reply:
My mistake. `ArgumentError` is best 😄